<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(217, 217, 217);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_6">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="4">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="imageComboBox1">
           <item>
            <property name="text">
             <string>FT Magnitude</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FT Phase</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FT Real</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FT Imaginary</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="insideButton_1">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Inside</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="outsideButton_1">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Outside</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="3">
        <widget class="QGraphicsView" name="imageComponent1"/>
       </item>
       <item row="2" column="1">
        <widget class="QGraphicsView" name="imageLabel1"/>
       </item>
       <item row="2" column="0">
        <widget class="QSlider" name="verticalSlider_1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Image #01</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="4">
     <widget class="QFrame" name="frame_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>650</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Output Mixer   </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>230</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Components</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="11" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Image #01</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QComboBox" name="comboBox_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Output #01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Output #02</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="radioButton2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Real/Imaginary</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_17">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Image #02</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" rowspan="3">
           <widget class="QComboBox" name="comboBox_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Magnitude</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Phase</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="label_19">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Image #04</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" rowspan="2">
           <widget class="QComboBox" name="comboBox_1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Magnitude</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Phase</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="13" column="1">
           <widget class="QComboBox" name="comboBox_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Magnitude</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Phase</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="10" column="1" rowspan="2">
           <widget class="QComboBox" name="comboBox_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Magnitude</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Phase</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Magnitude/Phase</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_18">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Image #03</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QFrame" name="frame_7">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QGraphicsView" name="outputImage1"/>
          </item>
          <item row="1" column="0">
           <widget class="QGraphicsView" name="outputImage2"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>100</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="4">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="imageComboBox2">
           <item>
            <property name="text">
             <string>FT Magnitude</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FT Phase</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FT Real</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FT Imaginary</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="insideButton_2">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Inside</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="outsideButton_2">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Outside</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="verticalSlider_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGraphicsView" name="imageLabel2"/>
       </item>
       <item row="1" column="3">
        <widget class="QGraphicsView" name="imageComponent2"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Image #02</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="verticalSlider">
      <property name="minimum">
       <number>50</number>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QGraphicsView" name="imageLabel3"/>
       </item>
       <item row="1" column="4">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="imageComboBox3">
           <item>
            <property name="text">
             <string>FT Magnitude</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FT Phase</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FT Real</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FT Imaginary</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="insideButton_3">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Inside</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="outsideButton_3">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Outside</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="verticalSlider_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QGraphicsView" name="imageComponent3"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Image #03</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QGraphicsView" name="imageLabel4"/>
       </item>
       <item row="1" column="4">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="imageComboBox4">
           <item>
            <property name="text">
             <string>FT Magnitude</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FT Phase</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FT Real</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FT Imaginary</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="insideButton_4">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Inside</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="outsideButton_4">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Outside</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="verticalSlider_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QGraphicsView" name="imageComponent4"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Image #04</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
